blocks/cards/cards.css

```diff
@@ -2,22 +2,112 @@
   list-style: none;
   margin: 0;
   padding: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(278px, 1fr));
-  grid-gap: 16px;
+  display: flex;
+  overflow-x: scroll;
+}
+
+@media (width > 600px) {
```

Comment:
Please correct breakpoints
/** Breakpoints
Mobile - 360 to 767
Tablet - 768 to 1279
Desktop – 1280 to 1920
Beyond 1920 the page will be centre aligned
*/

Reply:
Thanks @mkbansal1 .
Done.

Reply:
For SEO review comments:
since this site won't be going on public internet, could skipping the SEO comments for now.
Lighthout scrore of the PR is : 100